src/main/default/lwc/provideDetailsScreen/provideDetailsScreen.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-undef */
+import { LightningElement, api } from 'lwc';
+import {
+  FlowNavigationNextEvent,
+  FlowAttributeChangeEvent,
+  FlowNavigationBackEvent
+} from 'lightning/flowSupport';
+import {
+  registerComponentForInit,
+  initializeWithHeadless
+} from 'c/quanticHeadlessLoader';
+import next from '@salesforce/label/c.cookbook_Next';
+import previous from '@salesforce/label/c.cookbook_Previous';
+import provideDetailsTitle from '@salesforce/label/c.cookbook_ProvideDetailsTitle';
+import provideDetailsSubtitle from '@salesforce/label/c.cookbook_ProvideDetailsSubtitle';
+import priority from '@salesforce/label/c.cookbook_PriorityLabel';
+import origin from '@salesforce/label/c.cookbook_OriginLabel';
+import moreOptions from '@salesforce/label/c.cookbook_MoreOptions';
+
+/** @typedef {import("coveo").CaseAssistEngine} CaseAssistEngine */
+
+export default class ProvideDetailsScreen extends LightningElement {
+  labels = {
+    next,
+    previous,
+    priority,
+    origin,
+    provideDetailsTitle,
+    provideDetailsSubtitle,
+    moreOptions
+  };
+  /**
+   * availableActions is an array that contains the available flow actions when this component is used within a flow
+   * @see https://developer.salesforce.com/docs/component-library/bundle/lightning-flow-support/documentation
+   */
+  @api availableActions = [];
+  /**
+   * The ID of the engine instance the component registers to.
+   * @type {string}
+   */
+  @api engineId;
+  /**
+   * A stringified object representing the current fields set on the case.
+   */
+  @api caseData;
+
+  /** @type {CaseAssistEngine} */
+  engine;
+  /** @type{object} */
+  theCase;
+  /** @type {string} */
+  priority = '';
+  /** @type {string} */
+  origin = '';
+
+  connectedCallback() {
+    registerComponentForInit(this, this.engineId);
+    try {
+      this._caseData = JSON.parse(this.caseData);
```

Comment:
`this._caseData` does not seem to be defined.

Reply:
It would be nice to declare `_caseData` in the component with a proper JSDoc comment.